feature-libs/cart/saved-cart/components/page-context/saved-carts-page/new-saved-cart-order-entries.context.ts

```diff
@@ -39,6 +41,23 @@ export class NewSavedCartOrderEntriesContext implements AddOrderEntriesContext {
     savedCartInfo?: { name: string; description: string }
   ): Observable<ProductImportInfo> {
     return this.add(products, savedCartInfo).pipe(
+      tap((cartId: string) => {
+        this.importInfoService
+          .getResults(cartId)
+          .pipe(
+            take(products.length),
+            every(
+              (productInfo: ProductImportInfo) =>
+                productInfo.statusCode ===
+                ProductImportStatus.UNKNOWN_IDENTIFIER
+            )
+          )
+          .subscribe((isInvalid: boolean) => {
+            if (isInvalid) {
+              this.savedCartService.deleteSavedCart(cartId);
```

Comment:
When we try to import a csv with an invalid product, we get two errors:
<img width="1644" alt="image" src="https://user-images.githubusercontent.com/34665674/163463647-af138032-271e-4ba5-8caa-6b7d8ffe9885.png">

Can we prevent the cart not found error? Ideally preventing the creation of the cart rather than deleting.

Reply:
Hi, thanks for the review!

AFAIK, in order to get the information on whether the import was successful or not, you need to create a cart and pass the ID of that during the OCC call, so we're not sure if we can prevent the creation of that beforehand. This was also our initial idea, as to not have this redundant step, but no luck. If you're aware of a way (perhaps another OCC endpoint that should be used), please let us know.

Reply:
Yes we currently do not have an OCC endpoint for this. This solution should be fine for now.